### PR TITLE
feat(piaware): bump piaware/dump1090 to v5.0

### PR DESCRIPTION
**Pull request by Thom-x**

### CODE REVIEW

**Comment by Thom-x** [comment]
@deploy

**Comment by Srokap** [comment]
Looks like there is [version 5.0 of piaware](https://github.com/flightaware/piaware/tree/v5.0), but in https://github.com/flightaware/piaware_builder/commit/61fdc23afb5fbcfd2d9e36da709a1ef17917114b they did not make a tag yet.

**Comment by Srokap** [comment]
@Thom-x piaware_builder has v5.0 tag now https://github.com/flightaware/piaware_builder/tree/v5.0

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN git clone -b ${DUMP1090_VERSION} --depth 1 https://github.com/flightaware/du
 FROM debian:buster as piaware
 
 ENV DEBIAN_VERSION buster
-ENV PIAWARE_VERSION v3.8.1
+ENV PIAWARE_VERSION v4.0
```
Comment (Srokap) [inline Dockerfile:30]:
```suggestion
ENV PIAWARE_VERSION v5.0
```

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM debian:buster as dump1090
 
-ENV DUMP1090_VERSION v3.8.1
+ENV DUMP1090_VERSION v4.0
```
Comment (Srokap) [inline Dockerfile:3]:
```suggestion
ENV DUMP1090_VERSION v5.0
```
Version 5.0 is tagged now https://github.com/flightaware/dump1090/commit/f862b530c8cea51f4bb9e72356579be67818dcaf

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
@deploy

**Comment by Srokap** [comment]
FWIW I am running `thomx/fr24feed-piaware:PR-48` pulled 20min ago and it connects to both flightradar and flightaware with no problems. No aircraft around me at this hour, so no data, but map is working and it detects my RTL-SDR dongle
```
rtlsdr: using device #0: Generic RTL2832U (Generic, RTL2832U, 
```

**Comment by Thom-x** [comment]
Yep, same for me 👍

**Pull request merged**